modules/@apostrophecms/ui/ui/apos/components/AposTreeRows.vue

```diff
@@ -36,7 +36,23 @@
         >
           <drag-icon
             v-if="draggable && index === 0" class="apos-tree__row__handle"
-            size="20"
+            :size="20"
+            :fill-color="null"
```

Comment:
Does `fill-color` need to be bound if the values don't change?

Reply:
I think this was required because otherwise it's interpreted as a string?